src/sandstorm/util.c++

```diff
@@ -940,4 +940,13 @@ void TwoPartyServerWithClientBootstrap::taskFailed(kj::Exception&& exception) {
   KJ_LOG(ERROR, exception);
 }
 
+kj::String bytesToString(kj::ArrayPtr<const kj::byte> bytes) {
```

Comment:
I think this is equivalent to:

    kj::str(bytes.asChars())

Reply:
Good call; I'd missed `.asChars()`. Pushed  patch that uses that instead.